docs/source/releases/1.16.3/bug-fix-correct-return-value-from-output-file-list-check.yaml

```diff
@@ -0,0 +1,50 @@
+bug fix:
+- title: 'Return correct value from check output file list'
+  description: |
+    Previously did NOT return exit code from check_output_file_list, but relied on
+    exit_state being set.  exit_state did not seem to actually be set even when
+    sourcing this script, so just exit appropriately
+  files:
+    deleted:
+    - ''
+    moved:
+    - ''
+    added:
+    - ''
+    modified:
+    - 'tests/utils/check_output_file_list.sh'
+  date:
+    start: 2025-08-20
+    finish: 2025-08-21
+- title: 'Print return codes from each test script in global_terminator_satzen.sh example script'
```

Comment:
This now only runs one dataset at a time (with the incoming update), and each dataset is called from the pytest script directly,  so this will be a deprecated comment with the incoming patch